libpod/oci_conmon_linux.go

```diff
@@ -1318,6 +1318,10 @@ func (r *ConmonOCIRuntime) configureConmonEnv(ctr *Container, runtimeDir string)
 			env = append(env, e)
 		}
 	}
+	conf := os.Getenv("CONTAINERS_CONF")
+	if conf != "" {
+		env = append(env, fmt.Sprintf("CONTAINERS_CONF=%s", conf))
+	}
```

Comment:
```suggestion
	conf, ok := os.LookupEnv("CONTAINERS_CONF")
	if ok {
		env = append(env, fmt.Sprintf("CONTAINERS_CONF=%s", conf))
	}
```
We should also support an empty containers conf env var.

Reply:
Makes sense, I've folded that in.